File: java/src/juegoTexto/ProgramaPrincipal.java
package juegoTexto;

import java.util.ArrayList;
import civitas.CivitasJuego;

public class ProgramaPrincipal{
    public static void main (String[] args){
        ArrayList<String> nombres=new ArrayList<String>();
        nombres.add("Andrés");
        nombres.add("Carlos");
        nombres.add("Chema");
        nombres.add("Esmeralda");

        CivitasJuego juego=new CivitasJuego(nombres);
        VistaTextual vista=new VistaTextual();
        Controlador controlador=new Controlador(juego, vista);

        controlador.juega();
    }
}

File: ruby/lib/programa_principal.rb
# encoding:utf-8
require_relative('vista_textual')
require_relative('civitas_juego')
require_relative('dado')
require_relative('controlador')


module Civitas

    class ProgramaPrincipal
        def initialize
        end

        def self.main
            vista = Vista_textual.new
            juego = CivitasJuego.new(["Java", "Ruby"])
            Dado.instance.setDebug(true)
            controlador = Controlador.new(juego, vista)

            controlador.juega
        end

    end

    ProgramaPrincipal.main

end
